Group/domain/group.js

```diff
@@ -0,0 +1,34 @@
+const Group = require('../data/models/group');
+const create = async ({ name, user, details, image,price,discount,size }) => {
+    if (!name || !price || !discount || !size) {
+        throw new Error('Missing required fields');
+    }
+    const existingGroup = await Group.findOne({ where: { name } });
+    if (existingGroup) {
+        throw new Error('name already exists');
+    }
+    const user = await User.findOne({ where: { userEmail:user } });
```

Comment:
You should use auth service instead, please see auth service documentation, or ask me :)